### PR TITLE
Create languages summary in wagtail 

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

**Comment by saeedmb8** [comment]
screen shot 2016-03-22 at 8 29 39 pm

**Comment by saeedmb8** [comment]
@miltontony ready for review :)

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/admin/site_languages_summary.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% raw %}
+{% load i18n wagtailadmin_tags %}
+
+{% for summary in summaries %}
+  <li class="icon icon-doc-empty-inverse">
+    {% if summary.total > 1 %}
+      <span>{{ summary.total }}</span>{{ summary.language }} Pages
```
Comment (miltontony) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/admin/site_languages_summary.html]: You should use [Django's pluralize filter](https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#pluralize)

File: molo/core/wagtail_hooks.py [hunk 2]
```diff
@@ -65,3 +65,23 @@ def register_import_menu_item():
...
+    order = 500
+    template = 'admin/site_languages_summary.html'
+
+    def get_context(self):
+        languages = SiteLanguage.objects.all()
+        return {
+            'summaries': [{
+                'language': l.get_locale_display(),
+                'total': Page.objects.filter(
+                    languages__language__id=l.id, live=True).count()
```
Comment (miltontony) [inline molo/core/wagtail_hooks.py]: I don't think we need `live=True` here. I think we should count all pages, including drafts

**Comment by miltontony** [comment]
:+1: once travis

**Pull request merged**